packages/mobile/e2e/src/usecases/WalletConnectV1.js

```diff
@@ -335,6 +335,9 @@ export default WalletConnect = () => {
   })
 
   afterAll(async () => {
+    await device.reloadReactNative()
```

Comment:
Reload RN can create some weird state issues on the native side.
Can we instead relaunch the whole app?